src/Tracker/index.js

```diff
@@ -23,10 +23,9 @@ const integratorTrackedEvents = [
     'screen_face_selfie_intro',
     'screen_face_selfie_confirmation',
     'screen_face_video_intro',
-    'face_video_capture_step_1',
-    'face_video_capture_step_2',
-    'Starting_upload',
-    'Completed upload',
```

Comment:
I've not seen the RFC but I take it it was agreed there to remove the `Completed upload` event?

Reply:
The RFC was written to match the event names of the Android and IOS SDKs, so I had to play a bit of a matching game with the events sent by this SDK. Since the other SDKs only gave one upload event for when the upload started, I figured `Starting Upload` was a better match than `Completed upload` when I went back to map the names to match.